src/brpc/rdma/rdma_endpoint.cpp

```diff
@@ -71,20 +71,20 @@ static const size_t RESERVED_WR_NUM = 3;
 // message length (2B)
 // hello version (2B)
 // impl version (2B): 0 means should use tcp
-// block size (2B)
+// block size (4B)
 // sq size (2B)
 // rq size (2B)
 // GID (16B)
 // QP number (4B)
 static const char* MAGIC_STR = "RDMA";
 static const size_t MAGIC_STR_LEN = 4;
-static const size_t HELLO_MSG_LEN_MIN = 38;
+static const size_t HELLO_MSG_LEN_MIN = 40;
 static const size_t HELLO_MSG_LEN_MAX = 4096;
 static const size_t ACK_MSG_LEN = 4;
-static uint16_t g_rdma_hello_msg_len = 38;  // In Byte
+static uint16_t g_rdma_hello_msg_len = 40;  // In Byte
 static uint16_t g_rdma_hello_version = 1;
```

Comment:
Since we modify the hello message format, we should change the hello version.

Reply:
Done.